tests/test_utils.py

```diff
@@ -316,6 +340,159 @@ def check_multistream_classic(fnames):
 
 
+def setup_subdirs_and_fill_in_global_param_driver_match_test(
+        dict_s, result_basedir, state_basedir, test_data_dir):
+    ''' Fill in global parameter output directories for multiple driver runs
+        for driver-match testing
+
+    Parameters
+    ----------
+    dict_s: <dict of string.Template>
+        A dict of template of the global param file to be filled in
+        Keys: driver name
+    result_basedir: <str>
+        Base directory of output fluxes results; runs with different number of
+        processors are output to subdirectories under the base directory
+    state_basedir: <str>
+        Base directory of output state results; runs with different number of
+        processors are output to subdirectories under the base directory
+    test_data_dir: <str>
+        Base directory of test data
+
+    Returns
+    ----------
+    dict_global_param: <dict>
+        A dict of global parameter strings to be run with parameters filled in
+
+    Require
+    ----------
+    os
+    '''
+
+    dict_global_param = {}
+    for driver in dict_s.keys():
+        # Set up subdirectories for results and states
+        result_dir = os.path.join(result_basedir, driver)
+        state_dir = os.path.join(state_basedir, driver)
+        os.makedirs(result_dir, exist_ok=True)
+        os.makedirs(state_dir, exist_ok=True)
+
+        # Fill in global parameter options
+        s = dict_s[driver]
+        dict_global_param[driver] = s.safe_substitute(
+                test_data_dir=test_data_dir,
+                result_dir=result_dir,
+                state_dir=state_dir)
+
+    return(dict_global_param)
+
+
+def parse_classic_driver_outfile_name(fname):
+    '''helper function to parse VIC classic driver output file name'''
+    resultdir, filename = os.path.split(fname)
+    prefix, suffix = os.path.splitext(filename)
+    pieces = prefix.split('_')
+    lat, lon = map(float, pieces[-2:])
+    return VICOutFile(resultdir, prefix, lat, lon, suffix)
+
+
+def check_drivers_match_fluxes(list_drivers, result_basedir):
+    ''' Check whether the flux results are similar cross multiple drivers
+
+    Parameters
+    ----------
+    list_drivers: <dict>
+        A list of driver names to be compared
+        e.g., ['classic'; 'image']
+        NOTE: must have classic driver; classic driver will be the base for
+        comparison
+    result_basedir: <str>
+        Base directory of output fluxes results; results for drivers are
+        subdirectories under the base directory
+
+    Require
+    ----------
+    glob
+    xarray
+    numpy
+    collections.namedtuple
+    parse_classic_driver_outfile_name
+    VICOutFile
+    read_vic_ascii
+    '''
+
+    # Identify all classic driver output flux files
+    try:
+        list_fnames_classic = glob.glob(os.path.join(
+                                    result_basedir, 'classic', '*'))
+    except:
+        print('Error: must have classic driver for driver-match test!')
+        raise
```

Comment:
you also may want to say that there wasn't any output from the classic driver where you thought there would be..., that is the error you are actually raising.